base/levenshtein_dfa.cpp

```diff
@@ -13,7 +13,7 @@ namespace strings
 {
 namespace
 {
-inline size_t AbsDiff(size_t a, size_t b) { return a > b ? a - b : b - a; }
+size_t AbsDiff(size_t const a, size_t const b) { return a > b ? a - b : b - a; }
```

Comment:
Why const here? Was suggested by some checker?

Reply:
> Was suggested by some checker?

Yes.

Reply:
I mean, is there any logic behind this concrete function?
Could you please show the message of a checker?

We have thousands of functions in the codebase with integral input parameters without const.